asidi-bsas.md

```diff
@@ -0,0 +1,16 @@
+# Buenos Aires, Arg
+
+### Restaurants
+
+- McDonalds
+
```

Comment:
Buenos Aires has great places to eat Pizza, Empanadas, Pasta and of course Beef...
Now they might be expensive, but it would be nice if you add them too 😉